tests/test_cpo_trainer.py

```diff
@@ -37,15 +37,16 @@ def setUp(self):
 
     @parameterized.expand(
         [
-            ["gpt2", "sigmoid"],
-            ["t5", "hinge"],
-            ["gpt2", "ipo"],
-            ["t5", "ipo"],
-            ["gpt2", "simpo"],
-            ["t5", "simpo"],
+            ["gpt2", "sigmoid", "standard_preference"],
```

Comment:
At some point we should change this to replace `gpt2` with small instruct variants like `Qwen2.5-0.5B-Instruct` and `Llama-3.2-1B-Instruct` (although the latter is gated which means we can skip it). Applying CPO and other DPO-like methods to base models doesn't make sense IMO

Reply:
#2177